akka-javasdk-maven/akka-javasdk-parent/pom.xml

```diff
@@ -381,6 +380,9 @@
             <plugin>
                 <groupId>io.fabric8</groupId>
                 <artifactId>docker-maven-plugin</artifactId>
+                <configuration>
+                    <skip>${skip.docker}</skip>
+                </configuration>
```

Comment:
why was this added, or was it a merge conflict with standalone? we have all other conf in the pluginManagement section

Reply:
Yes, merge conflict, I must have resolved it backwards. I thought I kept your profile changes.

Reply:
sorted in [eaf2e41](https://github.com/akka/akka-sdk/pull/340/commits/eaf2e41df7f59fbee46d26cb3522b43739ad0375)